Mage.Sets/src/mage/cards/s/StriderRangerOfTheNorth.java

```diff
@@ -0,0 +1,96 @@
+package mage.cards.s;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.LandfallAbility;
+import mage.abilities.effects.ContinuousEffect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.continuous.BoostTargetEffect;
+import mage.abilities.effects.common.continuous.GainAbilityTargetEffect;
+import mage.abilities.keyword.FirstStrikeAbility;
+import mage.constants.*;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.target.common.TargetCreaturePermanent;
+import mage.target.targetpointer.FixedTarget;
+
+/**
+ *
+ * @author TiagoMDG
+ */
+public final class StriderRangerOfTheNorth extends CardImpl {
+
+    public StriderRangerOfTheNorth(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{2}{R}{G}");
+        
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.HUMAN);
+        this.subtype.add(SubType.RANGER);
+        this.power = new MageInt(4);
+        this.toughness = new MageInt(4);
+
+        // Landfall -- Whenever a land enters the battlefield under your control,
+        // target creature gets +1/+1 until end of turn.
+        // Then if that creature has power 4 or greater, it gains first strike until end of turn.
+
+        LandfallAbility ability = new LandfallAbility(
+                new StriderRangerOfTheNorthEffect(), false
+        );
+
+        ability.addTarget(new TargetCreaturePermanent());
+        this.addAbility(ability);
+    }
+
+    private StriderRangerOfTheNorth(final StriderRangerOfTheNorth card) {
+        super(card);
+    }
+
+    @Override
+    public StriderRangerOfTheNorth copy() {
+        return new StriderRangerOfTheNorth(this);
+    }
+}
+
+class StriderRangerOfTheNorthEffect extends OneShotEffect {
+
+    public StriderRangerOfTheNorthEffect() {
```

Comment:
nit, these constructors in the custom effect class don't need an access modifier (i.e. remove "public")